src/lib.rs

```diff
@@ -70,6 +70,12 @@ impl HidApiLock {
         {
             // Initialize the HID and prevent other HIDs from being created
             unsafe {
+                #[cfg(target_os = "openbsd")]
+                if ffi::hidapi_hid_init() == -1 {
```

Comment:
Instead of copying this block, either create an unsafe wrapper function around the hid_init functions, or alternatively keep only *hid_init*, and use https://doc.rust-lang.org/reference/items/external-blocks.html#the-link_name-attribute with *cfg_attr* to just rename the symbol. Then you do not need to use *cfg* here.

Reply:
Done. Diff looks much cleaner now.